elfo-dumper/src/actor.rs

```diff
@@ -106,7 +134,7 @@ impl Dumper {
                     let config = self.ctx.config();
                     self.interval.set_period(config.write_interval);
 
-                    path = config.path(self.ctx.key());
+                    self.render_path(&mut path);
                     self.file_registry
```

Comment:
If the active file is changed between reconfiguration and the following `DumpingTick`, we leave the previous one unclosed.

It seems we should simply remove `fn open()` here and always open files lazily